### PR TITLE
React navigaton

### PR DESCRIPTION
**Pull request by PatrykKaczmarek**
Following PR contains setup of basic navigation across the application. I've added 2 buttons on the home screen which lead to `AddEvent` screen and `EventInfo` screen. Moreover I've created ready to edit screen files.

### CODE REVIEW

File: app/screens/AddEventScreen/AddEventScreen.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { View } from 'react-native'
+import styles from './AddEventScreen.styles'
+
+const {
+  containerStyle
+} = styles
+
+class AddEventScreen extends React.Component {
```
Comment (Hellhiem) [inline app/screens/AddEventScreen/AddEventScreen.js]: import React, { Component } from 'react' and than use extends Component without react

File: app/screens/AddEventScreen/AddEventScreen.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  static navigationOptions = {
```
Comment (Hellhiem) [inline app/screens/AddEventScreen/AddEventScreen.js]: AddEventScreen.navigatorOptions {
title: 'Add Event'
};

File: app/screens/AddEventScreen/AddEventScreen.styles.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const styles = {
+  containerStyle: {
+    flex: 1,
+    backgroundColor: 'blue'
```
Comment (Hellhiem) [inline app/screens/AddEventScreen/AddEventScreen.styles.js:4]: you can store colors in some config files

File: app/screens/EventInfoScreen/EventInfoScreen.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { View } from 'react-native'
+import styles from './EventInfoScreen.styles'
+
+const {
+  containerStyle
+} = styles
+
+class EventInfoScreen extends React.Component {
+  static navigationOptions = {
```
Comment (Hellhiem) [inline app/screens/EventInfoScreen/EventInfoScreen.js]: as above

File: app/screens/EventInfoScreen/EventInfoScreen.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    title: 'Event Info'
```
Comment (Hellhiem) [inline app/screens/EventInfoScreen/EventInfoScreen.js]: you can create some messages file to store all messages in app

File: app/screens/EventInfoScreen/EventInfoScreen.styles.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+const styles = {
+  containerStyle: {
+    flex: 1,
+    backgroundColor: 'red'
```
Comment (Hellhiem) [inline app/screens/EventInfoScreen/EventInfoScreen.styles.js]: as above with colors

File: app/screens/HomeScreen/HomeScreen.js [hunk 7]
```diff
@@ -8,34 +9,69 @@ import styles from './HomeScreen.styles'
...
-      >
-        <CurrentWeatherInfo
-          headerInfo='Monday'
-          imageUrl='https://www.freeiconspng.com/uploads/weather-icon-png-16.png'
-          footerInfo='25 *C'
-        />
-      </View>
+class HomeScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Home'
```
Comment (Hellhiem) [inline app/screens/HomeScreen/HomeScreen.js]: as above with messages and navigator options

File: app/screens/HomeScreen/HomeScreen.js [hunk 8]
```diff
@@ -8,34 +9,69 @@ import styles from './HomeScreen.styles'
...
-      <View
-        style={currentWeatherContainerStyle}
-      >
-        <CurrentWeatherInfo
-          headerInfo='Monday'
-          imageUrl='https://www.freeiconspng.com/uploads/weather-icon-png-16.png'
-          footerInfo='25 *C'
-        />
-      </View>
+class HomeScreen extends React.Component {
```
Comment (Hellhiem) [inline app/screens/HomeScreen/HomeScreen.js]: same as above

**Review by Hellhiem — CHANGES_REQUESTED** [review]

File: app/screens/EventInfoScreen/EventInfoScreen.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { View } from 'react-native'
+// import styles from './EventInfoScreen.styles'
+
+const EventInfoScreen = () => {
```
Comment (drbonzo) [inline app/screens/EventInfoScreen/EventInfoScreen.js]: I think we need `DayInfoScreen` - that lists all events for given day

**Pull request merged**